src/hotspot/share/opto/machnode.hpp

```diff
@@ -135,6 +135,14 @@ class MachOper : public ResourceObj {
     return ::as_VectorSRegister(reg(ra_, node, idx));
   }
 #endif
+#if defined(S390)
+  VectorRegister as_VectorRegister(PhaseRegAlloc *ra_, const Node *node) const {
+    return ::as_VectorRegister(reg(ra_, node));
+  }
+  VectorRegister as_VectorRegister(PhaseRegAlloc *ra_, const Node *node, int idx) const {
+    return ::as_VectorRegister(reg(ra_, node, idx));
+ }
+#endif
 #if defined(AARCH64)
```

Comment:
why? no other platform does that.

Reply:
will there be any issue if we do:
```
#elif defined(AARCH64)
```
instead of 
```
#endif
#if defined(AARCH64)
```

The above one seems a bit better.